src/collections/_documentation/guides/manage-event-stream/index.md

```diff
@@ -0,0 +1,213 @@
+
+- **Accepted**: events processed and persisted displayed in your event and issue streams.
+- **Rate Limited**: events that Sentry threw away due to quotas being hit
+- **Filtered**: events that were blocked based on your inbound filter rules.
+
+![STATS View]({% asset guides/manage-event-stream/12.png @path %})
+
+Clicking on a project name will open the project settings view where you can manage the project's Inbound Filtering and Rate Limiting.
+
+### > **What are my busiest projects?**
+
+The [Discover](https://docs.sentry.io/workflow/discover/) view provides a flexible query builder to query event data cross-project.
+
+- Open the Discover view from the left-side navigation bar
+- From the top-level filter bar select `View All Projects` from the project drop-down and apply the desired date range
+- Build, Run, and Save the following query
+
+    ![Busiest Projects]({% asset guides/manage-event-stream/16.png @path %})
+
+> NOTE: Queries in Discover are executed against a subset of events defined in the **top-level filter bar** based on selected _projects_ and _date range_. The Projects drop-down allows you to search for events across
+>
+> - **My Projects:** all projects that you are assigned to via team-membership within the organization
+> - **All Projects:** all projects within the organization (requires owner permission role)
+> - **Subset** of selected projects
+>
+>   ![Project Filter]({% asset guides/manage-event-stream/17.png @path %})
+
+### > **What issues are consuming my quota?**
+
+- Navigate to the Discover view
+- From the top-level filter bar select `View All Projects` from the project drop-down and apply the desired date range
+- Build, Run, and Save the following query
+- Notice that you can open an issue details page by clicking on the `issue id` link
+
+    ![Busiest Issues]({% asset guides/manage-event-stream/10.png @path %})
+
+- Alternatively, you can sort the `Issues` table by issue `Frequency`
+
+    ![Issue Frequency]({% asset guides/manage-event-stream/13.png @path %})
+
+### > **How to set proper rate limits?**
+
+A good way to set a project rate limit is by figuring out the expected event volume based on past traffic. Let's look at an example
+
+![Calculating rate limits]({% asset guides/manage-event-stream/14.png @path %})
+
+- Open the project DSN key configuration under `[Project Settings] > Client Keys > [Configure]`
+- Take a look at the `KEY USAGE IN THE LAST 30 DAYS` graph. Max daily rate in the last month is < 326K
+- Based on that, we can define a ceiling **daily** max value of ~330K which is ~13,750 events an **hour**.
+- Notice that you can set a daily, hourly or minute-based rate limit. We'd recommend using an hourly rate to avoid situations where a random event spike might exhaust your daily set quota and leave you blind for the rest of the day.
```

Comment:
```suggestion
- Notice that you can set a daily, hourly, or minute-based rate limit. We'd recommend using an hourly rate to avoid situations where a random event spike might exhaust your daily set quota and leave you blind for the rest of the day.
```